app/base/templates/base-site.html

```diff
@@ -55,7 +55,8 @@
         </div>
       </div>
 
-      {% include 'site_template/footer.html' %}
```

Comment:
Why the footer is removed? Do we even have a footer @saifulkhan? Can you comment on this?

Reply:
If it does not cause any issue, then keep it.